lib/archethic_web/controllers/api/web_hosting_controller.ex

```diff
@@ -383,5 +405,24 @@ defmodule ArchethicWeb.API.WebHostingController do
     end
   end
 
-  defp get_file_content(_, _, _), do: :file_error
+  defp do_get_file_content(_, _), do: :file_error
+
+  @spec search_transaction(binary()) :: {:ok, Transaction.t()} | {:error, atom()}
+  defp search_transaction(address) do
+    # :cache_tx is started by the ArchethicWeb.Supervisor
+    case LRU.get(:cache_tx, address) do
+      nil ->
+        case Archethic.search_transaction(address) do
+          {:ok, tx} ->
+            LRU.put(:cache_tx, address, tx)
```

Comment:
Actually the web hosting controller require only 3 fields from a transaction: address, data.content, validation_stamp.timestamp
Maybe we can reduce the data stored in the LRU by creating a Transaction object with only theses fieds and remove all other unecessary fields

Reply:
Using a tuple should do it.